src/WebJobs.Extensions.DurableTask/DefaultPlatformInformation.cs

```diff
@@ -10,28 +10,23 @@ namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
     /// using the DI-injected INameResolver.
     /// </summary>
 #pragma warning disable CS0612 // Type or member is obsolete
-    internal class DefaultPlatformInformationProvider : IPlatformInformationService
+    internal class DefaultPlatformInformation : IPlatformInformation
 #pragma warning restore CS0612 // Type or member is obsolete
     {
         private readonly INameResolver nameResolver;
 
-        public DefaultPlatformInformationProvider(INameResolver nameResolver)
+        public DefaultPlatformInformation(INameResolver nameResolver)
         {
             this.nameResolver = nameResolver;
         }
 
-        public bool InConsumption()
-        {
-            return this.InLinuxConsumption() | this.InWindowsConsumption();
-        }
-
-        public bool InWindowsConsumption()
+        private bool InWindowsConsumption()
         {
             string value = this.nameResolver.Resolve("WEBSITE_SKU");
             return string.Equals(value, "Dynamic", StringComparison.OrdinalIgnoreCase);
         }
 
-        public bool InLinuxConsumption()
+        private bool InLinuxConsumption()
         {
             string containerName = this.GetContainerName();
             string azureWebsiteInstanceId = this.nameResolver.Resolve("WEBSITE_INSTANCE_ID");
```

Comment:
Consider using `Lazy<string>` variables here so that each environment variable is only fetched once, regardless of how many times methods needing this environment variable are called.